src/ifElse.test.js

```diff
@@ -1,11 +1,53 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
```

Comment:
Replace the generic test description with a more descriptive one, for example: 'ifElse function should be declared and be an instance of Function'